static/app/views/alerts/builder/projectProvider.tsx

```diff
@@ -48,8 +50,12 @@ function AlertBuilderProjectProvider(props: Props) {
     }
 
     // fetch members list for mail action fields
-    fetchOrgMembers(api, organization.slug, [project.id]).then(mem => setMembers(mem));
-  }, [api, organization, project]);
+    fetchOrgMembers(api, organization.slug, [project.id]).then(mem => {
+      if (isMounted()) {
```

Comment:
instead of a function, shouldn't the `useIsMounted` hook return only `true` or `false`?

Reply:
returning the current ref would never get the updated value

Reply:
i agree that would be preferred though

Reply:
I switched it to returning the entire ref?